libtiledbvcf/src/cli/tiledbvcf.cc

```diff
@@ -481,6 +478,34 @@ void add_store(CLI::App& app) {
       "Enable TileDB stats for vcf header array usage");
 
   cmd->option_defaults()->group("Advanced options");
+  cmd->add_option(
+         "--ratio-tiledb-memory",
+         args->ratio_tiledb_memory,
+         "Ratio of memory budget allocated to TileDB::sm.mem.total_budget")
+      ->check(CLI::Range(0.01, 0.99));
+  cmd->add_option(
+      "--max-tiledb-memory-mb",
+      args->max_tiledb_memory_mb,
+      "Maximum memory allocated to TileDB::sm.mem.total_budget (MiB)");
+  cmd->add_option(
+      "--input-record-buffer-mb",
+      args->input_record_buffer_mb,
+      "Size of input record buffer for each sample file (MiB)");
+  cmd->add_option(
+         "--avg-bcf-record-size",
```

Comment:
Unless this is somehow specific to BCF files, I think we should use `vcf` in the arg/function names rather than `bcf`.

Reply:
Done.